src/ProductCard/StoreLocationButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from 'react'
+import ChevronRight from '../Icons/ChevronRight'
+import MapMarker from '../Icons/MapMarker'
+import clsx from 'clsx'
+
+import styles from './productCard.module.scss'
+
+interface Props {
+  storeLocation?: string
+  onClick?: Function
+}
+
+const StoreLocationButton: FunctionComponent<Props> = ({ storeLocation, onClick }) => {
+  const handleClick = () => {
+    if (onClick) {
+      onClick
+    }
+  }
+
+  const marker = storeLocation && <MapMarker fillColor="#d63426" className={styles.mapMarker} />
+  const arrow = storeLocation && <ChevronRight style={{ transform: 'rotate(90deg)', marginRight: '10px' }} />
+  const btnText = storeLocation ? <span className={styles.mobileColor}>Available to try in-store</span> : 'Online Only'
+
+  return (
+    <button className={clsx(styles['store-location-btn'], storeLocation && styles.align)} onClick={() => handleClick()}>
```

Comment:
Button needs a "type". If you don't specify a type, it's "submit" by default. You don't want that unless this is in a form. The default behavior is to refresh the page with type="submit" unless you do an event.preventDefault() which I see you do not have